quokka_editor_front/src/components/organisms/login/Login.tsx

```diff
@@ -13,8 +13,12 @@ const Login: React.FC = () => {
   const navigate = useNavigate();
 
   return (
-    <div className="flex flex-col justify-center items-center">
-      <h1 className="text-3xl font-bold">Login</h1>
+    <div className="flex flex-col justify-center items-center p-9">
+      <div className ="w-56 m-9 relative shadow rounded-[30px]">
+        <div className="top-0 left-0 absolute w-[115px] h-10 bg-gradient-to-r from-[#1f21292e] to-[#5F6066] rounded-[30px]"></div>
+            <button type="button" className="relative px-8 py-2 cursor-pointer bg-transparent border-none outline-none text-white">Log in</button>
+            <button type="button" className="relative px-7 py-2 cursor-pointer bg-transparent border-none outline-none text-[#ffffff29]">Register</button>
```

Comment:
wartości 'from' ,'to' oraz 'text' można dodać do configu tailwinda i z nich korzystać, żeby było czytelniej